editor/src/components/inspector/sections/component-section/variables-in-scope-utils.ts

```diff
@@ -391,7 +391,20 @@ export function useVariablesInScopeForSelectedElement(
       return []
     }
 
-    let variablesInScopeForSelectedPath = variablesInScope[EP.toString(selectedViewPath)]
+    let variablesInScopeForSelectedPath: VariableData | null = (() => {
```

Comment:
if `variablesInScope[EP.toString(selectedViewPath)]` is not null, then the should be identical to the one on master